<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convertbase</class>
 <widget class="QMainWindow" name="convertbase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>转换工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="m_sourceList"/>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="m_outputList"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenDir"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionConvert"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionOpenDir"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionConvert"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpenDir">
   <property name="icon">
    <iconset resource="convertools.qrc">
     <normaloff>:/res/open.png</normaloff>:/res/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="convertools.qrc">
     <normaloff>:/res/search.png</normaloff>:/res/search.png</iconset>
   </property>
   <property name="text">
    <string>查找文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="convertools.qrc">
     <normaloff>:/res/exit.png</normaloff>:/res/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="icon">
    <iconset resource="convertools.qrc">
     <normaloff>:/res/export.png</normaloff>:/res/export.png</iconset>
   </property>
   <property name="text">
    <string>GDF转BGF</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="convertools.qrc">
     <normaloff>:/res/open_file.png</normaloff>:/res/open_file.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="convertools.qrc">
     <normaloff>:/res/delete.png</normaloff>:/res/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="convertools.qrc"/>
 </resources>
 <connections/>
</ui>
